nni/algorithms/hpo/dngo_tuner/search_space.json

```diff
@@ -0,0 +1,6 @@
+{
+  "batch_size": {"_type":"choice", "_value": [16, 32, 64, 128]},
+  "hidden_size":{"_type":"choice","_value":[128, 256, 512, 1024]},
+  "lr":{"_type":"choice","_value":[0.0001, 0.001, 0.01, 0.1]},
+  "momentum":{"_type":"uniform","_value":[0, 1]}
```

Comment:
Does DNGO tuner works on string choices for example `["apple", "banana", "orange"]`.

Reply:
yes, technically it supports any string choices. But it's quite meaningless.

Reply:
But I don't think it's handled in your code.

Have you tested?